packaging/python/build_wheels.bash

```diff
@@ -75,7 +75,7 @@ build_wheel_linux() {
     echo " - Installing build requirements"
     #auditwheel needs to be installed with python3
     # see upstream WIP PR https://github.com/pypa/auditwheel/pull/368
-    pip install git+https://github.com/neuronsimulator/auditwheel.git@exclude_so_files
+    pip install git+https://github.com/neuronsimulator/auditwheel.git@exclude_so_files_patch
```

Comment:
Curious - does this rebase bring any known improvement?

Reply:
havent checked, was just a quick try

Reply:
to fix current CI issue
https://dev.azure.com/neuronsimulator/nrn/_build/results?buildId=6024&view=logs&j=55224220-a8bb-54b8-5bb0-36c069f4f094&t=bd86b240-67dd-5230-60ad-a330e5f7dcb8&l=353